test/e2e/validators.go

```diff
@@ -182,6 +182,9 @@ func validateController(
 		return testBuildRun.Status.Succeeded
 	}, 550*time.Second, 5*time.Second).Should(Equal(trueCondition), "BuildRun did not succeed")
 
+	completionTime, err := time.Parse(time.RFC3339Nano, testBuildRun.Status.CompletionTime.String())
+	startTime, err := time.Parse(time.RFC3339Nano, testBuildRun.Status.StartTime.String())
```

Comment:
Add `Expect(err).NotTo(HaveOccurred(), "Failed to pase completion/start time")` after each line.